### PR TITLE
Update website

### PR DESCRIPTION
**Pull request by JavierMatosD**
Modifies generateStars to capture https://github.com/kubernetes-client/c/

Still an issue where it tries to fetch stars for `https://gitlab.freedesktop.org/xorg/lib/libxpm`

However, the original URL is: "https://gitlab.freedesktop.org/xorg/lib/libxpm"  so it is unclear why it's trying to fetch stars here.

### CODE REVIEW

File: scripts/generatePackages.js [hunk 1]
```diff
@@ -40,7 +40,15 @@ async function readManifest(manifestFile) {
     let out = {};
     for (let key of Object.keys(parsed)) {
         if (key.startsWith("$")) continue;
-        out[makeManifestKeyReadable(key)] = parsed[key];
+        if (key.match("homepage"))
+        {
+            if (parsed[key].startsWith("https://") || parsed[key].startsWith("http://")){
+                out[key] = parsed[key];
+            }
+        }else
```
Comment (ras0219-msft) [inline scripts/generatePackages.js:48]: Formatting.

File: en/package/abseil.html [hunk 2]
```diff
@@ -354,7 +354,7 @@
                 <div class="sidebar-header">Homepage</div>
                 <div>
                     <img class="sidebar-logo" src="/assets/package-pages/house-solid.svg" alt="Homepage:">
-                    <a href="https://github.com/abseil/abseil-cpp">https://github.com/abseil/abseil-cpp</a>
+                    Unavailable
```
Comment (ras0219-msft) [inline en/package/abseil.html:357]: This seems incorrect. Was this intended?

**Review by ras0219-msft — COMMENTED** [review]
When spot checking ~8 or so packages I see they've _all_ had their valid homepage links replaced with "Unavailable". Is that correct?

**Comment by JavierMatosD** [comment]
> When spot checking ~8 or so packages I see they've _all_ had their valid homepage links replaced with "Unavailable". Is that correct?

It should be fixed now

**Review by ras0219-msft — APPROVED** [review]

**Pull request merged**